doc/changelog/03-notations/11172-master+coercion-notation-interleaved-printing.rst

```diff
@@ -0,0 +1 @@
+- In printing, now interleave search for notations and removal of coercions (`#11172 <https://github.com/coq/coq/pull/11172>`_, by Hugo Herbelin).
```

Comment:
What about:

```suggestion
- The printing algorithm now interleave search for notations and removal of coercions
  (`#11172 <https://github.com/coq/coq/pull/11172>`_, by Hugo Herbelin).
```

Reply:
And this time, I picked back the commit to my working copy!